src/rgw/rgw_rest_sts.h

```diff
@@ -89,6 +89,12 @@ class DefaultStrategy : public rgw::auth::Strategy,
     return s->info.args.get("WebIdentityToken");
   }
 
+  /* The method implements TokenExtractor. This method is not used by STS. */
+  std::string get_service_token(const req_state* const s) const override {
+    static std::string empty_val;
```

Comment:
The `static` deserves a comment at least. However, the Casey's idea on not even having the entire method at all is really appealing.

Reply:
This is refactored with Casey's idea in later commit, so not there anymore.